R/rdeephaven/R/client_wrapper.R

```diff
@@ -102,6 +105,10 @@ setMethod(
     if ((auth_token != "") && (auth_type == "anonymous")) {
       warning("'auth_token' was set but it will not be used, as 'auth_type' is 'anonymous'.")
     }
+    
+    if (((username != "") || (password != "")) && auth_type != "basic") {
+      warning("At least one of 'username' and 'password' were set but they will not be used, as 'auth_type' is not 'basic'.")
+    }
```

Comment:
This check is redundant with the test in the "basic" block above.  The test should be in the basic block because then it localizes the logic for "basic" stuff.